### PR TITLE
Update NFT-Marketplace-Part-1.md

### PR DESCRIPTION
**Pull request by yanukadeneth99**
- Removed 'Equal to' in 2 places
- Changed the Chainlist link to open the Celo Alfajores Testnet straight
- Changed the purchase code to use the '_to.call' method and moved the delete to the end of the code. (Please check this, and let me know if that's incorrect).
- Fixed Indentation for Shipping it
- Fixed Emoji icons for Next Steps
- Added Discord Invite to showcase channel link

### CODE REVIEW

**Review by haardikk21 — CHANGES_REQUESTED** [review]

File: NFT-Marketplace-Part-1.md [hunk 1]
```diff
@@ -597,10 +599,10 @@ NFT Marketplace deployed to: 0x9014DD98Cd14B26c76069356247cE6d762018220
 
 You're all set! You can look up your contract on [CeloScan](https://alfajores.celoscan.io/) as well.
 
-### 🫡 Next Steps
+### 🌟 Next Steps
 
 This level was all about the smart contract. Moving forward, we will develop the subgraph for this contract, and then finally develop the frontend for the dApp using Next.
 
-Drop your CeloScan link in the Discord `#showcase` channel and share your progress! See you in the next one 🫡
+Drop your CeloScan link in the Discord [#showcase](https://discord.gg/HPunW7rAKb) channel and share your progress! See you in the next one 🌟
```
[the remark this answers is not in the document]
Comment (yanukadeneth99) [reply NFT-Marketplace-Part-1.md:606]: Yup.

File: NFT-Marketplace-Part-1.md [hunk 2]
```diff
@@ -330,18 +330,19 @@ function purchaseListing(address nftAddress, uint256 tokenId)
...
     // Transfer NFT from seller to buyer
     IERC721(nftAddress).safeTransferFrom(
         listing.seller,
         msg.sender,
         tokenId
     );
 
-    // Transfer ETH sent by buyer to seller
-    payable(listing.seller).transfer(msg.value);
+    // Transfer ETH sent from buyer to seller
```
[the remark this answers is not in the document]
Comment (yanukadeneth99) [reply NFT-Marketplace-Part-1.md:340]: Sure, I'll look into that.

File: NFT-Marketplace-Part-1.md [hunk 3]
```diff
@@ -459,14 +460,15 @@ contract NFTMarketplace {
...
         IERC721(nftAddress).safeTransferFrom(
             listing.seller,
             msg.sender,
             tokenId
         );
-        payable(listing.seller).transfer(msg.value);
+        (bool sent, ) = payable(listing.seller).call{value: msg.value}("");
+        require(sent, "Failed to transfer eth");
+        
+        delete listings[nftAddress][tokenId];
```
[the remark this answers is not in the document]
Comment (yanukadeneth99) [reply NFT-Marketplace-Part-1.md:471]: You might be right. If the `safeTransferFrom` or `call` fails, then the deletion is reverted right?

[the remark this answers is not in the document]
Comment (yanukadeneth99) [reply NFT-Marketplace-Part-1.md]: Changed this back.

**Comment by yanukadeneth99** [comment]
Changed the delete code location in : https://github.com/LearnWeb3DAO/Celo-Track/pull/6/commits/57d528d995bd72845381e391155a00f73abd30fd

File: NFT-Marketplace-Part-1.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yanukadeneth99) [reply NFT-Marketplace-Part-1.md:343]: This is requested in PR : https://github.com/LearnWeb3DAO/Celo-Track/pull/7

**Review by haardikk21 — APPROVED** [review]

**Pull request merged**